### ISSUE TITLE
Update post flair

### ISSUE THREAD

**Issue by gastrodon** [turnage/graw#69]
Does this library have a way of modifying the flair of an existing post? I can't see anything in the code that would suggest as much with a peruse, though I wanted to find out if it was possible

**Comment by turnage** [comment]
There is no flair support in the API.

**Comment by gastrodon** [comment]
In the Reddit API, or this wrapper? I know that some other Reddit clients like Boost that use the api support flair adding, so there must be _some_ way to do that

**Comment by turnage** [comment]
The Reddit API supports it iirc, this does not.

Adding support for more calls is relatively simple: an interface needs to
be named to contain them, and the values need to be read/written through
JSON with Reddit.

Pull requests welcome.

**Comment by gastrodon** [comment]
Certainly, I'll get started on this when I have a minuted. I wanted to make sure that nobody else had already done it before I did
